content/faq/security/general.md

```diff
@@ -1,9 +1,88 @@
 ---
 description: Find the answers to common security related FAQs
-keywords: Docker, Docker Hub, Docker Desktop secuirty FAQs, secuirty, platform
-title: Security FAQs
+keywords: Docker, Docker Hub, Docker Desktop secuirty FAQs, secuirty, platform, Docker Scout, admin, security
+title: General security FAQs
 ---
 
-## How does Docker Desktop handle and store authentication information?
+### How do I report a vulnerability?
+
+If you’ve discovered a security vulnerability in Docker, we encourage you to report it responsibly. Report security issues to security@docker.com so that they can be quickly addressed by our team.
+
+### How are passwords managed when SSO isn't used? 
+
+Passwords are encrypted and salt-hashed. If you use application level passwords instead of SSO, you are responsible for ensuring that your employees know how to pick strong passwords, don't share passwords, and don't reuse passwords across multiple systems. 
```

Comment:
```suggestion
Passwords are encrypted and salt-hashed. If you use application-level passwords instead of SSO, you are responsible for ensuring that your employees know how to pick strong passwords, don't share passwords, and don't reuse passwords across multiple systems. 
```